.github/workflows/pr-tests.yml

```
@@ -0,0 +1,63 @@
+name: PR Documentation Tests
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  doc-test:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)